src/js/apis/c2c/DocumentService.js

```diff
@@ -7,6 +7,51 @@ DocumentService.prototype.getAll = function (params) {
   return this.api.get('/' + this.documentType + 's', { params });
 };
 
+DocumentService.prototype.fullDownload = function (params, limit, progress) {
+  // will load the ENTIRE list of document. Limited to 2000 docs
+  const this_ = this;
+
+  limit = limit || 2000;
```

Comment:
and so, if `limit` is set to 3000 then it is not restricted to 2000?

Reply:
Good catch!